cfgov/tccp/jinja2/tccp/cards.html

```diff
@@ -61,17 +61,27 @@ <h1>Card search results</h1>
             {'heading': 'Purchase APR'},
             {'heading': 'Availability'},
             {'heading': 'Account fee'},
-            {'heading': 'Secured'},
-            {'heading': 'Rewards'},
+            {'heading': 'Late payment fee'},
+            {'heading': 'First late fee'},
+            {'heading': 'Balance transfer APR'},
+            {'heading': 'Balance transfer fee'},
+            {'heading': 'Secured card'},
+            {'heading': 'Introductory APR offers'},
+            {'heading': 'Offers rewards'},
         ] %}
         {%- set card_rows = [] %}
         {%- for card in results %}
             {% do card_rows.append( [
                 ('<a href="' ~ card.url ~ '">' ~ card.product_name ~ '</a>') | safe,
                 ('%.2f%%' | format(card.purchase_apr * 100)) if card.purchase_apr else 'None',
-                ('Regionally: ' ~ card.state_limitations | join(', ')) if card.state_limitations else 'Nationally',
+                ('Regionally; ' ~ card.state_limitations | join(', ')) if card.state_limitations else 'Nationally',
```

Comment:
😮

Reply:
This is how it's specified in @caheberer's design!